node/src/types/deploy.rs

```diff
@@ -721,14 +725,11 @@ impl Deploy {
     /// Returns true if and only if:
     ///   * the chain_name is correct,
     ///   * the configured parameters are complied with,
-    ///   * the deploy is valid
-    ///
-    /// Note: if everything else checks out, calls the computationally expensive `is_valid` method.
-    pub fn is_acceptable(
+    pub fn is_config_compliant(
```

Comment:
I've noticed that we're not checking if deploy's associated keys are within the chainspec limit. I think we should add it to `is_config_compliant` as it's an easy and cheap check.

Reply:
Added  check